src/std/io/port.ss

```diff
@@ -0,0 +1,44 @@
+(import
+  :gerbil/gambit/ports
+  :std/sugar)
+
+(export raw-port raw-port-port)
+
+;; Raw wrapper for a port without any buffering
+(defstruct raw-port (port) final: #t unchecked: #t)
+
+(defrule (def-raw-port-method (name port . args) body ...)
+  (lambda (self . args)
+    (let (port (&raw-port-port self))
+      body ...)))
```

Comment:
did you forget a `defmethod` in here? this doesn't define methods, in fact it does nothing!